### PR TITLE
VZ 8883: Upgrade webhook error

### PR DESCRIPTION
**Pull request by anders-swanson**
Updates the webhook error on upgrade to be more user friendly.

### CODE REVIEW

File: platform-operator/apis/verrazzano/validators/validate.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func ValidateVersion(requestedVersion string) error {
 		return err
 	}
 	if !requestedSemVer.IsEqualTo(bomSemVer) {
-		return fmt.Errorf("Requested version %s does not match BOM version %s", requestedSemVer.ToString(), bomSemVer.ToString())
+		return fmt.Errorf("invalid version %s: Please use Verrazzano version %s", requestedSemVer.ToString(), bomSemVer.ToString())
```
Comment (rosemarymarano) [inline platform-operator/apis/verrazzano/validators/validate.go:105]: Any reason why invalid is lowercase (here and in the following error message)?

**Review by rosemarymarano — APPROVED** [review]
Otherwise, LGTM.

File: platform-operator/apis/verrazzano/validators/validate.go (hunk 1 above)
Comment (anders-swanson) [reply platform-operator/apis/verrazzano/validators/validate.go:105]: it's a style constraint for returning these types of error messages. Don't begin sentences with a capitalized letter, and don't end them with a full-stop.

Comment (rosemarymarano) [reply platform-operator/apis/verrazzano/validators/validate.go:105]: @anders-swanson ok, thanks for the explanation.

**Pull request merged**